engine/physics/src/physics/physics_2d.cpp

```diff
@@ -1044,6 +1044,7 @@ namespace dmPhysics
     {
         b2Shape* shape = (b2Shape*) _shape;
         shape->m_radius = radius * world->m_Context->m_Scale;
+        shape->m_creationScale = radius;
```

Comment:
Make sure we update `m_creationScale` (which is Defold specific modification of box2d) which used as a basic radius value for a  scale applying